src/views/bridge-confirmation/bridge-confirmation.view.tsx

```diff
@@ -77,102 +70,65 @@ const BridgeConfirmation: FC = () => {
-                if (!isFirstLoad) {
-                  setIsFadeVisible(false);
-                }
-                const isTokenEther = token.address === ethersConstants.AddressZero;
-                const remainder = isTokenEther ? amount.add(newFee).sub(tokenBalance) : undefined;
-                const isRemainderPositive = remainder ? !remainder.isNegative() : undefined;
-                const newMaxPossibleAmountConsideringFee =
-                  isTokenEther && remainder && isRemainderPositive ? amount.sub(remainder) : amount;
-
-                const msTimeout = FADE_DURATION_IN_SECONDS * 1000;
-                const etherToken = isTokenEther ? token : getEtherToken(from);
-
-                const hasOnScreenFeeChanged =
-                  oldFee !== undefined &&
-                  formatTokenAmount(oldFee, etherToken) !== formatTokenAmount(newFee, etherToken);
-
-                setShouldUpdateOnScreenFee(hasOnScreenFeeChanged);
-
-                setMaxPossibleAmountConsideringFee((oldMaxPossibleAmountConsideringFee) => {
-                  const formattedOldMaxPossibleAmountConsideringFee =
-                    oldMaxPossibleAmountConsideringFee &&
-                    (oldMaxPossibleAmountConsideringFee.isNegative()
-                      ? "0"
-                      : formatTokenAmount(oldMaxPossibleAmountConsideringFee, token));
-
-                  const formattedNewMaxPossibleAmountConsideringFee =
-                    newMaxPossibleAmountConsideringFee.isNegative()
-                      ? "0"
-                      : formatTokenAmount(newMaxPossibleAmountConsideringFee, token);
-
-                  const hasOnScreenAmountChanged =
-                    formattedOldMaxPossibleAmountConsideringFee !==
-                    formattedNewMaxPossibleAmountConsideringFee;
-
-                  setShouldUpdateOnScreenAmount(hasOnScreenAmountChanged);
-
-                  const areAmountsEqual =
-                    oldMaxPossibleAmountConsideringFee !== undefined &&
-                    oldMaxPossibleAmountConsideringFee.eq(newMaxPossibleAmountConsideringFee);
-
-                  if (!areAmountsEqual) {
-                    setTimeout(() => {
-                      setMaxPossibleAmountConsideringFee(newMaxPossibleAmountConsideringFee);
-                    }, msTimeout);
-                  }
-                  return oldMaxPossibleAmountConsideringFee;
-                });
-
-                setTimeout(() => {
-                  setEstimatedGas({
-                    status: "successful",
-                    data: newGas,
-                  });
-                  setIsFadeVisible(true);
-                }, msTimeout);
-
-                return oldGas;
+                return amount;
               }
+            })();
+
+            setMaxAmountConsideringFee((oldMaxAmountConsideringFee) => {
+              setShouldAmountFade(!oldMaxAmountConsideringFee?.eq(newMaxAmountConsideringFee));
+              return oldMaxAmountConsideringFee;
```

Comment:
`setShouldAmountFade(!maxAmountConsideringFee?.eq(newMaxAmountConsideringFee))` seems to work for me.